scripts/scil_mrds_select_number_of_tensors.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Fit the info from a NuFO map to the plausible MRDS solutions.
+
+Each MRDS input is a list of 5 files:
+    - Component size
```

Comment:
Not a fan of component size. You mean signal fraction of the component, is that it?